apps/coscrad-frontend/src/components/Header/Header.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import { screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { getDummyConfigurableContent } from '../../app/App.spec';
+import { renderWithProviders } from '../../utils/test-utils';
+import { Header } from './Header';
+
+describe('Header', () => {
+    it('should load configurable content', () => {
+        const dummyConfigurableContent = getDummyConfigurableContent();
+        const siteTitle = 'The New Website Title';
+        const subTitle = 'The New Website Subtitle';
+        const myConfigurableContent = { ...dummyConfigurableContent, siteTitle, subTitle };
+
+        const { baseElement } = renderWithProviders(
+            <MemoryRouter>
+                <Header />
+            </MemoryRouter>,
+            {
+                contentConfig: myConfigurableContent,
+            }
+        );
+
+        const searchPattern = new RegExp(siteTitle);
+        const screenRes1 = screen.getByText(searchPattern);
+        const searchPattern2 = new RegExp(subTitle);
+        const screenRes2 = screen.getByText(searchPattern2);
+        const screenRes = screenRes1 && screenRes2 ? true : false;
```

Comment:
Don't do it this way. Have two separate tests-
it should use `subTitle` configurable content
it should use `screenRes` configurable content

If you don't use one of these props in the component, a test should fail that tells you exactly which prop you failed to use.

Reply:
This would be another case where I wonder if it makes sense to write a helper function that loads the configurable content so it doesn't need to happen twice.